homeassistant/components/shelly/binary_sensor.py

```diff
@@ -123,3 +137,15 @@ class ShellyRestBinarySensor(ShellyRestAttributeEntity, BinarySensorEntity):
     def is_on(self):
         """Return true if REST sensor state is on."""
         return bool(self.attribute_value)
+
+
+class ShellySleepingBinarySensor(
+    ShellySleepingBlockAttributeEntity, BinarySensorEntity
+):
+    """Represent a shelly sleeping binary sensor."""
+
+    @property
+    def is_on(self):
+        """Return true if sensor state is on."""
+
+        return self.attribute_value in [STATE_ON, 1]
```

Comment:
Why would `attribute_value` be `STATE_ON` and not just a boolean ?

Reply:
When this is restored from `async_get_last_state` the restored state is `on` or `off`